CONTRIBUTING.md

```diff
@@ -3,12 +3,16 @@
 Contributions to this Module are very welcome! We follow a fairly standard [pull request 
 process](https://help.github.com/articles/about-pull-requests/) for contributions, subject to the following guidelines:
  
-1. [File a GitHub issue](#file-a-github-issue)
-1. [Update the documentation](#update-the-documentation)
-1. [Update the tests](#update-the-tests)
-1. [Update the code](#update-the-code)
-1. [Create a pull request](#create-a-pull-request)
-1. [Merge and release](#merge-and-release)
+- [Contribution Guidelines](#contribution-guidelines)
```

Comment:
What is this linking to?

Reply:
My editor was automatically forammting and adding this